src/advancedpicker/MobiScroll.js

```diff
@@ -270,21 +233,23 @@ dojo.declare("advancedpicker.MobiScroll", mobile.widget._Widget, {
             var date = parseInt(this._mxObject.get(this.dateAttr), 10);
```

Comment:
This parsing of int (as data is a number of ms since 1970. is not really necessary as current Mendix will return a real number. Though this code will not break. It can be left in.